estuary/utils/story.py

```diff
@@ -144,6 +145,51 @@ def get_sibling_nodes_count(self, results, reverse=False):
             return correlated_nodes[::-1]
         return correlated_nodes
 
+    def get_wait_times(self, results):
+        """
+        Get the wait time between two different artifacts.
+
+        :param list results: contains inflated results from Neo4j
+        :return: list of wait time tuples in order of the pipeline (oldest to newest)
+        :rtype: list
```

Comment:
Could you add the following to the docstring (not sure if it's `raise` or `raises`)?
```python
:raises RuntimeError: if results has less than 2 elements
```